NEXT_CHANGELOG.md

```diff
@@ -5,7 +5,11 @@
 ### New Features and Improvements
 
  * Add support for `power_bi_task` in jobs ([#4647](https://github.com/databricks/terraform-provider-databricks/pull/4647))
+<<<<<<< HEAD
```

Comment:
we need to review merge conflict

Reply:
Ah new laptop/editor, need to setup autosave.